products/zap/bbls/metadata.yml

```diff
@@ -0,0 +1,120 @@
+name: bbls
+display_name: Zoning Application Portal (ZAP) - BBL
+summary: "The Department of City Planning (DCP) processes land use applications submitted\
+  \ by City or other public agencies and other (private) applicants. This data set\
+  \ provides information on land use applications, specifically the project tracking\
+  \ and description data related to approximately 30,000 projects since the late 1970\u2019\
+  s. \r\n\r\nZAP project data appears on NYC Planning - Zoning Application Portal\
+  \ Search. It includes data migrated from the prior applications tracking system\
+  \ (LUCATS) and covers all projects that have been \u2018Noticed\u2019 (that they\
+  \ will appear before the City Planning Commission in 30 or more days for Certification\
+  \ as complete for the ULURP process to begin) or Filed (a CEQR or Land Use application\
+  \ has been formally submitted to the Department for review) through completion (approval,\
+  \ disapproval, withdrawal, or termination). For more information on the land use\
+  \ and environmental review application process see:\r\nhttps://www1.nyc.gov/site/planning/applicants/applicant-portal/application-process.page\r\
+  \n\r\nYou can explore this data in the Zoning Application Portal: https://zap.planning.nyc.gov/projects\r\
+  \n\r\nTo report data errors or for questions, email <a href=\"mailto:zap_feedback_dl@planning.nyc.gov\"\
+  >zap_feedback_dl@planning.nyc.gov </a>."
+description: "The Department of City Planning (DCP) processes land use applications\
```

Comment:
@damonmcc Let's format the long lines with nice yaml style. [this](https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines) might be helpful. 

TLDR: remove carriage-returns and newlines, `\u2019` chars, and backslashes. Also, we should just remove the summary from our models. I can't remember quite what it was for, but it's not used, and now just an annoyance.